sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBase.java

```diff
@@ -7378,6 +7381,18 @@ public boolean getIconResumed() throws SdlException {
 		return _iconResumed;
 	}
 
+	/**
+	 * Method to retrieve the RegisterAppInterface Response message that was sent back from the
+	 * module. It contains various attributes about the connected module and can be used to adapt
+	 * to different module types and their supported features.
+	 *
+	 * @return RegisterAppInterfaceResponse received from the module or null if the app has not yet
+	 * registered with the module.
+	 */
+	public RegisterAppInterfaceResponse getRegisterAppInterfaceResponse(){

```

Comment:
is this info available after `SdlManagerListener.onStart()`? Do I need to trigger it in any way?

Reply:
Yes it is available after the manager starts. For other questions please use the slack channel.